src/utils/__tests__/resolve_url.test.ts

```diff
@@ -27,19 +27,19 @@ describe("utils - resolveURL", () => {
     expect(resolveURL("http://toto.com/a"))
       .toBe("http://toto.com/a");
     expect(resolveURL("http://toto.com/a", "b/c/d/", "g.a"))
-      .toBe("http://toto.com/a/b/c/d/g.a");
+      .toBe("http://toto.com/b/c/d/g.a");
   });
```

Comment:
there is only relative path here, so there should be concatenated.  Expected result `"http://toto.com/a/b/c/d/g.a"` is correct